### PR TITLE
Allow extra meta/option boxes to be added in the pod-edit screens

### PR DESCRIPTION
**Pull request by JoryHogeveen**
Handy to add meta/option boxes to the side of the pod edit page

### CODE REVIEW

**Comment by sc0ttkclark** [comment]
I wonder how much effort it would be to support meta box functions for this purpose

**Comment by JoryHogeveen** [comment]
What do you mean? The form is around the whole page so these inputs should be available in the post values right?

**Comment by JoryHogeveen** [comment]
Oh, you mean like regular post pages so we can use `add_meta_box()` for this?

**Comment by JoryHogeveen** [comment]
Works great! :)
In this version only the 'side' context is available because of the tabs (which already have action for adding extra tabs).

**Comment by JoryHogeveen** [comment]
TODO:

- [ ] Allow registration of settings with meta boxes (currently the value's aren't being saved because Pods doesn't know whem I guess)
- [ ] Tests!

**Comment by JoryHogeveen** [comment]
Little but of doc info:

When using the add_meta_box function on Pods edit pages the metabox callback content should be wrapped in a element with the class `pods-submittable-fields` otherwise Pods will not save the data.

```
<div class="pods-submittable-fields">
// Metabox callback content
</div>
```

File: ui/admin/setup-edit.php [hunk 1]
```diff
@@ -134,6 +134,9 @@
     if ( empty( $field_type_fields ) )
         $no_additional[] = $field_type;
 }
+
+// Make use of the WP core meta box functionality
+do_action( 'add_meta_boxes', 'pods_edit', $pod );
```
Comment (sc0ttkclark) [inline ui/admin/setup-edit.php]: Needs inline hook docs https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters

File: ui/admin/setup-edit.php [hunk 2]
```diff
@@ -875,6 +878,7 @@
                 </div>
             </div>
             <!-- /#submitdiv -->
+            <?php do_meta_boxes( 'pods_edit', 'side', $pod ); ?>
```
Comment (sc0ttkclark) [inline ui/admin/setup-edit.php]: Let's wrap this in a `pods-submittable-fields` div just to make things easier

Comment (JoryHogeveen) [inline ui/admin/setup-edit.php]: Smart thinking!

Comment (JoryHogeveen) [inline ui/admin/setup-edit.php]: Tested and works, see https://github.com/pods-framework/pods/pull/3609/commits/5c2be5fb768a52e426822fcf3c0b1993f1b460a9

**Comment by JoryHogeveen** [comment]
Looks good for merging to me. Anyone?

**Pull request closed without merging**